design-editor/package-lock.json

```diff
@@ -530,11 +530,11 @@
       "integrity": "sha1-Cr9PHKpbyx96nYrMbepPqqBLrJs="
     },
     "fs-extra": {
```

Comment:
do we really want to add package-lock. As far as I can remember we've decided not to add it unless we make changes in package.json

Reply:
Is removed